Sources/SwiftFormat/Core/Parsing.swift

```diff
@@ -65,9 +65,7 @@ func parseAndEmitDiagnostics(
 
       // Downgrade editor placeholders to warnings, because it is useful to support formatting
```

Comment:
Please update this comment to indicate that editor placeholder errors are being ignored instead of downgraded to warnings.

(Which I think is fine. Editor placeholders are common in source being interactively edited and formatted that I think it's fine to not report them at all.)